index.html

```diff
@@ -261,45 +261,17 @@ <h3>Partnerships</h3>
-
-                                <h4><a href="./news/2019-08-12-Coding-Prototype-Co-design.html">Co-design at Summer Coding Camps</a></h4>
-                                <time class="floe-date" datetime="2019-08-12">August 12, 2019</time>
-                                <p>
-                                    Floe participated in Summer Coding Camps where students used a coding environment prototype,
-                                    markers, paint and robots to create artworks.
-                                </p>
-                                <p><a href="./news/2019-08-12-Coding-Prototype-Co-design.html">Read more: Co-design at Summer Coding Camps</a></p>
-
-                                <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the University of Guelph Accessibility Conference</a></h4>
-                                <time class="floe-date" datetime="2019-07-02">July 2, 2019</time>
-                                <p>
-                                    On May 28 and 29, Floe made an appearance at the 2019 <a href="https://opened.uoguelph.ca/accessibility-conference">Accessibility Conference</a> put on by <a href="https://opened.uoguelph.ca/">Open Learning and Educational Support</a> at the <a href="https://opened.uoguelph.ca/">University of Guelph</a>.
-                                </p>
-                                <p><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Read more: Floe at the University of Guelph Accessibility Conference</a></p>
+                                <ul>
+                                    <li><time class="floe-date" datetime="2019-11-05">July 15, 2020</time>: <a href="2020-07-15-Platform-Co-op-Update.html">Collaborative Co-Design with Cooperatives</a></h4></li>
+                                    <li><time class="floe-date" datetime="2019-11-05">July 14, 2020</time>: <a href="2020-07-14-New-Resources-from-Coding-to-Learn.html">New Resources from Coding to Learn</a></li>
+                                    <li><time class="floe-date" datetime="2020-07-02">July 2, 2020</time>: <a href="2020-07-02-WeCountUpdate.html">We Count Update: Inclusion in the Data Economy</a></li>
+                                    <li><time class="floe-date" datetime="2020-07-02">June 28, 2020</time>: <a href="2020-06-28-AIHEC-Update.html">American Indian Higher Education Consortium Using Floe Tools and Resources</a></li>
+                                    <li><time class="floe-date" datetime="2020-05-20">May 20, 2020</time>: <a href="2020-05-20-DEEP-Update.html">Webinar with Dr. Toon Calders on July 9, 2020: Machine Learning: Bias in, Bias out</a></li>
+                                    <li><time class="floe-date" datetime="2020-03-27">March 27, 2020</time>: <a href="2020-03-27-ODAO-Update.html">Updates from Our Doors are Open</a></li>
+                                </ul>
```

Comment:
The hrefs are all pointing to the wrong URLs. They should start with "news/"

Reply:
Another issue is that the contrast is too low for all of the links in the grey box. Any chance we can bump this up a bit along with this change?

Reply:
I was hoping to avoid any changes to the CSS in this PR. 

I plan to address CSS when we implement the new design features. Maybe move away from Foundation at the same time.